server.js

```diff
@@ -9,17 +9,35 @@ const server = express();
 // allow server to parse JSON bodies from POST/PUT/DELETE requests
 server.use(bodyParser.json());
 
+// TODO: DONE - write your server code here
 
+// get Bear model
+const Bear = require('./models');
 
+// POST /bears -- accepts bear
+server.post('/bears', (req, res) => {
+  const newBear = new Bear(req.body);
+  newBear.save()
+    .catch(error => res.status(500).json({ error }))
```

Comment:
Typically I see catch statements last. and honestly, in the world of promises/async code, I didn't know you could even do this. Catch be for a Then. hahaha